### PR TITLE
[probably shouldn't merge] Two possible fixes for the egg runtime

### PR DESCRIPTION
**Pull request by Wizardcrying**
The first fix makes the variable a temporary var.
The second fix checks the variable to see if it is null, and if it is, returns. Much hackier.

### CODE REVIEW

**Comment by PJB3005** [comment]
Temp vars don't do anything in procs, they're only useful in class definitions.

File: code/modules/mob/living/simple_animal/borer/egg.dm [hunk 1]
```diff
@@ -72,7 +72,9 @@
 
 
 /obj/item/weapon/reagent_containers/food/snacks/borer_egg/process()
-	var/turf/location = get_turf(src)
+	var/tmp/turf/location = get_turf(src)
+	if(location == null)
```
Comment (PJB3005) [inline code/modules/mob/living/simple_animal/borer/egg.dm]: Could just do `!location`.

Comment (Wizardcrying) [inline code/modules/mob/living/simple_animal/borer/egg.dm]: Right.

**Comment by clusterfack** [comment]
The cause of this runtime is most likely the qdeletion of the egg, setting the loc to null. 

This is due to the fact it lacks a destroy proc to remove it from processing objects upon deletion, it should also null the reference to the recruiter as well. Please make that change as well.

**Pull request merged**